### PR TITLE
Contact-form

### PR DESCRIPTION
**Pull request by Adanzeakonobi**
This is a Personal Portfolio Project.

The following were updated.
- Updated index.html
- Updated style.css
- Updated readme.md

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,213 +2,260 @@
...
+        <img class="social-icon" src="./icons/angel.png" alt="Angel"/>
+      </a>
+      <a href="#"  class="social-link">
+        <img class="social-icon" src="./icons/twitter.png" alt="Twitter"/>
+      </a>
+      <a href="#"  class="social-link">
+        <img class="social-icon" src="./icons/medium.png" alt="Medium"/>
+      </a>
+    </footer>
+  </form>
```
Comment (TedLivist) [inline index.html]: - [ ] Let's try to be more organized in our code and obey the semantic principles. The `footer` tag should wrap the `form`, not the other way round. Kindly refactor accordingly. Ensure the same design is followed.

- [ ] The social media icons are not part of the form hence they should not be wrapped by the `form` tag. Kindly move the icons to be below the `form` element.

File: index.html [hunk 2]
```diff
@@ -2,213 +2,260 @@
...
+            <li class="tech">Gitlab</li>
+            <li class="tech">Terminal</li>
+          </ul>
+        </div>
+      </section>
+      <section class="contact">
+        <form action="https://formspree.io/f/xlezlkyw" method="post">
+          <p class="form-detail">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+             <input type="text" placeholder="Full Name" maxlength="30" required>
+             <input type="email" placeholder="Email address" required>
```
Comment (TedLivist) [inline index.html:235]: - [ ] Your **Full Name** and **Email** inputs are missing the bottom-border styling with have in the Figma design. Kindly add them.

- [ ] In keeping with the design, I would also recommend you remove the black borders of the message `textarea`.

  Your Implementation | Figma Design
  :---:|:---:
  adanze-1 | adanze-figma

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Adanzeakonobi,

Good job so far!
✔️ Good documentation 
✔️ Nice design
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by prolajumokeoni — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

all-done-gif

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**